src/components/login/signUp.tsx

```diff
@@ -1,57 +1,147 @@
-import './login.css';
 import { createUserWithEmailAndPassword } from 'firebase/auth';
-import { useState } from 'react';
 import { auth } from '../../firebase-config';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
 
-function SignUp() {
-    const [email, setEmail] = useState('');
-    const [password, setPassword] = useState('');
+function Copyright(props: any) {
```

Comment:
Do not use `any` if you can avoid it. In this case, it's `TypographyProps` that should be used since you're spreading any props that come into the `Copyright` component into `<Typography />`. If you're not sure what type it is... Just use ChatGTP. Super fast way to find it.

For example, 
`function Copyright(props: TypographyProps) {`

Reply:
Also, don't forget to updated the following import:
`import Typography, { TypographyProps } from '@mui/material/Typography';`